topiary-queries/queries/nickel.scm

```diff
@@ -249,15 +249,6 @@
 
 ;; Annotations
 
-; We want the equals sign to be aligned with the annotations, if they
-; coexist, so create a scope that covers them both.
-(
-  (#scope_id! "annotated_assignment")
-  (annot) @prepend_begin_scope
-  .
-  "=" @append_end_scope
-)
-
```

Comment:
So, this is just not needed at all to ensure what the coments say, that `=` and `|` are aligned? Does this impact in anyway the content of the field if it's e.g. multiline (what's after the `=`)?

Reply:
Right; I think when I put this together the first time, I had the belief that a scope was necessary. However, on reflection, this doesn't seem to be the case and -- indeed -- the scope is too liberal.

The input-output tests pass -- which include the Nickel stdlib -- so AFAIK, this change doesn't have a deleterious effect elsewhere.